stm32-modules/heater-shaker/firmware/host_comms_task/freertos_comms_task.cpp

```diff
@@ -101,7 +101,7 @@ extern __ALIGN_BEGIN uint8_t
     USBD_CDC_CfgFSDesc[USB_CDC_CONFIG_DESC_SIZ] __ALIGN_END;
 }
 
-__IO ITStatus UartReady = SET;
+static bool UartReady = true;
```

Comment:
Using `std::atomic_bool` here might be useful, I think there's a chance that caching issues could pop up since this is set in an interrupt